examples/data-sources/kentik-synthetics_agent-list/run.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
Are you sure that casual users, current Kentik customers are not going to mess up their Kentik config by running the example? We should at minimum warn about changes (if any) that the execution is going to make to their Kentik config. I assume that the example is harmless as of now, because the provider currently cannot make any changes, but keep in mind that Kentik customers pay for every synthetic test they run, so if the example creates tests (in the future) running this script may cost them money. So, we must not forget to cleanup after a test run.
Overall, as much as I do understand that it is useful to have functional example executing against the "real API",  I'm afraid this script make become dangerous in the future. It is very easy to add more stuff to example terraform resource files and forget that a naive Kentik customer may run it against their live configuration.